cilium-cli/connectivity/check/features.go

```diff
@@ -323,40 +323,46 @@ func (ct *ConnectivityTest) detectFeatures(ctx context.Context) error {
 	if cm.Data == nil {
 		return fmt.Errorf("ConfigMap %q does not contain any configuration", defaults.ConfigMapName)
 	}
+
```

Comment:
nit: I'd add the PR description to the commit itself so that we have direct context in the git history. Feel free to address this only in case from other reviews we'd need to rerun CI.